content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -7,7 +7,6 @@ metaDescription: Prisma Client features that are currently in preview.
 
 The following [Preview](/about/prisma/releases#preview) feature flags are available for Prisma Client and the Prisma schema:
 
-- [`interactiveTransactions`](/concepts/components/prisma-client/transactions#interactive-transactions-in-preview) since release [2.29.0](https://github.com/prisma/prisma/releases/tag/2.29.0) - [Submit feedback](https://github.com/prisma/prisma/issues/8664)
 - [`fullTextSearch`](/concepts/components/prisma-client/full-text-search) since release [2.30.0](https://github.com/prisma/prisma/releases/tag/2.30.0) - [Submit feedback](https://github.com/prisma/prisma/issues/8877)
 - [`referentialIntegrity`](/concepts/components/prisma-schema/relations/relation-mode) since release [3.1.1](https://github.com/prisma/prisma/releases/tag/3.1.1) - [Submit feedback](https://github.com/prisma/prisma/issues/9380)
```

Comment:
Heads up that this preview feature is also in GA in 4.7.0, so it will need to be added to the new section you created below as well in https://github.com/prisma/docs/pull/4117 — coordinate with @keerlu about this